test/e2e/edge_test.go

```diff
@@ -14,7 +17,7 @@ import (
 )
 
 func depositFromL1(ctx context.Context, opsman *operations.Manager, t *testing.T) {
-	amount := new(big.Int).SetUint64(250000000000000000)
+	amount := new(big.Int).SetUint64(2500000000000000000)
```

Comment:
Be careful in the latest version of the smc we limit deposits to 0.25 eth. Maybe the latest version of the network is not updated

Reply:
OK